packages/chat/package.json

```diff
@@ -22,16 +23,14 @@
   },
   "files": [
     "es/**/*",
-    "custom-elements.json",
-    "telemetry.yml"
+    "custom-elements.json"
```

Comment:
please change back, telemetry was added to collect data on who's using the chat package